scripts/scratch/00-populate-deploy-artifact-from-env.js

```diff
@@ -0,0 +1,44 @@
+const { ZERO_ADDRESS } = require('@aragon/contract-helpers-test')
+const runOrWrapScript = require('../helpers/run-or-wrap-script')
+const { readNetworkState, persistNetworkState } = require('../helpers/persisted-network-state')
+
+const DEPLOYER = process.env.DEPLOYER
+const CHAIN_ID = parseInt(process.env.CHAIN_ID)
```

Comment:
Actually, we can check the provided `CHAIN_ID` against the real one using the `eth_chainid` RPC call

Reply:
I've to conclusion it is excess. Can take if from RPC call to record in the state file. The checks for correspondence of the chainId will be done automatically anyway for live networks for which chainId is specified in hardhat.config.js